app/Filament/Subcontractor/Resources/ChantierAssignmentsResource.php

```
@@ -0,0 +1,153 @@
+                    ->color(fn (int $state): string => match (true) {
+                        $state >= 100 => 'success',
+                        $state > 0 => 'warning',
+                        default => 'danger',
+                    }),
+                Tables\Columns\TextColumn::make('financial_progress')
+                    ->label('Progression financière')
+                    ->getStateUsing(function (Chantier $record) {
+                        $user = auth()->user();
+                        $subcontractorId = $user->contact->thirdParty->id;
+
+                        $orderTotal = PurchaseOrder::where('chantier_id', $record->id)
```

PHPStan (2.2.7)

[error] Call to an undefined static method App\Models\Commerce\PurchaseOrder::where().


(staticMethod.notFound)